pom.xml

```diff
@@ -176,6 +176,7 @@
         <jboss-slf4j.version>1.2.1.Final</jboss-slf4j.version>
         <keycloak.version>11.0.3</keycloak.version>
         <httpclient.version>4.5.13</httpclient.version>
+        <apicurio-common-rest-client.version>0.0.2.Final</apicurio-common-rest-client.version>
```

Comment:
@riprasad FYI Carles is introducing a new shared library that eventually will need to be built in PNC.  :)